vpc/service/assign_addresses_v3.go

```diff
@@ -913,7 +913,7 @@ func (vpcService *vpcService) assignIPsToENI(ctx context.Context, req *vpcapi.As
 		}
 	}
 
-	_, err = tx.ExecContext(ctx, "UPDATE branch_enis SET last_assigned_to = now() WHERE branch_eni = $1", aws.StringValue(iface.NetworkInterfaceId))
+	_, err = tx.ExecContext(ctx, "UPDATE branch_enis SET last_assigned_to = now(), last_used = now() WHERE branch_eni = $1", aws.StringValue(iface.NetworkInterfaceId))
```

Comment:
Can you say a little bit more about how you expect `last_assigned_to` and `last_used` to differ?